### PR TITLE
Integrate form and simulation modules

### PR DESCRIPTION
**Pull request by PabloTercerosCC**
# Pull request title

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: form/button_module.py [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
             name = name_entry.get()
-            print(f"{name_label}", name)
+            form_data[name_label[:-1]] = name
 
-        selected_option = option.get()
-        selected_percentage = percentage_var.get()
-        print("Selected Option:", selected_option)
-        print("Selected Percentage:", selected_percentage)
+        form_data["Selected Option"] = option.get()
+        form_data["Selected Percentage"] = percentage_var.get()
```
Comment (luisarce02) [inline form/button_module.py]: Move this to a constants

File: form/form_module.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from tkinter import ttk
+from .entry_module import create_name_entry_frame
+from .radio_button_module import create_radio_button_frame
+from .drop_menu_module import create_drop_menu_frame
+from .button_module import create_button_frame, form_built_flag
+import json
+
+def build_form(callback):
+    root = tk.Tk()
+    root.title("AntOps Form")
```
Comment (luisarce02) [inline form/form_module.py]: move to a constants

File: form/form_module.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    style = ttk.Style()
+    style.theme_use('clam')
```
Comment (luisarce02) [inline form/form_module.py]: move to a constants

File: form/form_module.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+    def create_title_label(parent, text, row, column=0):
+        label = tk.Label(parent, text=text, font=('Helvetica', 12, 'bold'))
```
Comment (luisarce02) [inline form/form_module.py]: same here and others

File: simulator/simulator.py [hunk 5]
```diff
@@ -53,7 +55,7 @@ def simulator():
                 draw_peripheral_pointer(pointer_position, WHITE)
             else:
                 draw_mouse_pointer(pointer_position, WHITE)
-            draw_shoots(red_points, RED, peripheral)
+            draw_shoots(red_points, GREEN if data['Calibre'] == '1' else BLUE, peripheral)
```
Comment (luisarce02) [inline simulator/simulator.py:58]: to a constants

**Review by luisarce02 — CHANGES_REQUESTED** [review]

File: simulator/simulator.py (hunk 5 above)
Comment (PabloTercerosCC) [reply simulator/simulator.py:58]: Not constants

Comment (luisarce02) [reply simulator/simulator.py:58]: mark as a test field with a comment or in a constant file

**Review by luisarce02 — APPROVED** [review]
LGTM

**Pull request merged**